feature/gnmi/ate_tests/telemetry_port_speed_test/telemetry_port_speed_test.go

```diff
@@ -192,35 +193,43 @@ func (tc *testCase) configureDUT(t *testing.T) {
 		iPath := d.Interface(iName)
 		fptest.LogQuery(t, port.String(), iPath.Config(), i)
 		gnmi.Replace(t, tc.dut, iPath.Config(), i)
-	}
-
-	if tc.lagType == lagTypeLACP {
-		lacp := &oc.Lacp_Interface{Name: ygot.String(tc.aggID)}
-		lacp.LacpMode = oc.Lacp_LacpActivityType_ACTIVE
-
-		lacpPath := d.Lacp().Interface(tc.aggID)
-		fptest.LogQuery(t, "LACP", lacpPath.Config(), lacp)
-		gnmi.Replace(t, tc.dut, lacpPath.Config(), lacp)
-		t.Cleanup(func() {
-			gnmi.Delete(t, tc.dut, lacpPath.Config())
-		})
+		if *deviations.ExplicitPortSpeed {
+			fptest.SetPortSpeed(t, port)
+		}
 	}
 
 	agg := &oc.Interface{Name: ygot.String(tc.aggID)}
 	tc.configAggregateDUT(agg, &dutIPs)
 	aggPath := d.Interface(tc.aggID)
 	fptest.LogQuery(t, tc.aggID, aggPath.Config(), agg)
 	gnmi.Replace(t, tc.dut, aggPath.Config(), agg)
+	if *deviations.ExplicitInterfaceInDefaultVRF {
+		fptest.AssignToNetworkInstance(t, tc.dut, tc.aggID, *deviations.DefaultNetworkInstance, 0)
+	}
 	t.Cleanup(func() {
 		gnmi.Delete(t, tc.dut, gnmi.OC().Interface(tc.aggID).Aggregation().MinLinks().Config())
 		for _, port := range tc.dutPorts {
 			iName := port.Name()
 			iPath := d.Interface(iName)
 			gnmi.Replace(t, tc.dut, iPath.Config(), &oc.Interface{Name: ygot.String(iName), Type: ethernetCsmacd})
 		}
+		if *deviations.ExplicitInterfaceInDefaultVRF {
+			gnmi.Delete(t, tc.dut, gnmi.OC().NetworkInstance(*deviations.DefaultNetworkInstance).Interface(tc.aggID+".0").Config())
+		}
 		gnmi.Delete(t, tc.dut, aggPath.Config())
 	})
 
+	if tc.lagType == lagTypeLACP {
+		lacp := &oc.Lacp_Interface{Name: ygot.String(tc.aggID)}
+		lacp.LacpMode = oc.Lacp_LacpActivityType_ACTIVE
+
+		lacpPath := d.Lacp().Interface(tc.aggID)
+		fptest.LogQuery(t, "LACP", lacpPath.Config(), lacp)
+		gnmi.Replace(t, tc.dut, lacpPath.Config(), lacp)
+		t.Cleanup(func() {
+			gnmi.Delete(t, tc.dut, lacpPath.Config())
+		})
+	}
```

Comment:
as we discussed offline, other vendors might have dependency of LACP-lag-interface on /lacp/ path configuration. please consider a way to have alternate config/delete order.

Reply:
@jjing1122 I pushed the commit, where I am restoring the original config order. Also, introducing `gnmi.BatchDelete` for both LACP and lag-interface config based on `deviations.AggregateAtomicUpdate`, considering this same deviation was used to configure both LACP and lag-interface together.